src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.service;
+
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.model.User;
+
+public interface AuthenticationService {
+    User login(String email, String password) throws AuthenticationException;
+
+    void register(String email, String password) throws RegistrationException;
```

Comment:
According to the task in README.md method `register` should return `User` object